### ISSUE TITLE
more than one schema

### ISSUE THREAD

**Issue by yathe** [THUDM/GATNE#41]
hi
How to set up if there is more than one schema between two nodes？
How can the model be used to calculate the correlation of nodes？

**Comment by cenyk1230** [comment]
Hi @yathe,

1. The current version of code doesn't support more than one schema. We will add this feature soon. 

2. You can compute the dot product between nodes as the correlation of nodes.

**Comment by yathe** [comment]
thank you !
If I want to apply this model to a recommendation system, but the training data (social science data) does not score the recommended results, how can I evaluate the model ,can you provide some ideas?

**Comment by cenyk1230** [comment]
You can use the validation set to evaluate the model.

**Comment by yathe** [comment]
Actually, in my application scenario,  There is no score for recommendation results in the dataset (social science data) , I haven't thought of a good way to evaluate the model😭
thank you very much！！
